TransformationSystem/Client/TaskManager.py

```diff
@@ -333,6 +335,116 @@ def __init__( self, transClient = None, logger = None, submissionClient = None,
     self.destinationPlugin_o = None
 
   def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '' ):
+
+    bulkFlag = self.opsH.getValue( "Transformations/BulkSubmission", False )
+    if bulkFlag:
+      result = self.__prepareTransformationTasksBulk( transBody, taskDict, owner, ownerGroup, ownerDN )
+    else:
+      result = self.__prepareTransformationTasks( transBody, taskDict, owner, ownerGroup, ownerDN )
```

Comment:
Do not use private methods here please. There is also a unit test I restored in https://github.com/fstagni/DIRAC/blob/fixTestsv6r14/TransformationSystem/Client/test/Test_Client_TransformationSystem.py (PR: https://github.com/DIRACGrid/DIRAC/pull/2883/) that tests `prepareTransformationTasks` (what becomes here `__prepareTransformationTasks`) that should be kept.